CMakeLists.txt

```diff
@@ -1235,34 +1214,26 @@ if(BUILD_PAT AND APPLE)
   install(DIRECTORY "${PROJECT_BINARY_DIR}/${RADIANCE_PATH}/usr/local/radiance/" DESTINATION ./ParametricAnalysisTool.app/Contents/Resources/Radiance/ COMPONENT PAT USE_SOURCE_PERMISSIONS)
 endif()
 
-export(TARGETS
```

Comment:
Will need to test this carefully, but I think the only export we need from OpenStudio is the openstudio.dll and the cli. Specifically I don't think we should export every intermediate library.